src/components/ReleaseService/ReleasePlan/TriggerRelease/AddIssueSection/AddIssueSection.tsx

```diff
@@ -0,0 +1,224 @@
+              className="pf-v5-u-mb-0 pf-v5-u-pb-0 pf-v5-u-pl-0"
+            >
+              <ToolbarContent>
+                <ToolbarGroup align={{ default: 'alignLeft' }}>
+                  <ToolbarItem className="pf-v5-u-ml-0">
+                    <SearchInput
+                      name="nameInput"
+                      data-test={`${field}-input-filter`}
+                      type="search"
+                      aria-label="name filter"
+                      placeholder="Filter by name..."
+                      onChange={(e, n) => onNameInput(n)}
+                      value={nameFilter}
+                    />
+                  </ToolbarItem>
+                  <ToolbarItem>
+                    <AddIssueModal bugArrayHelper={addNewBug} issueType={issueType} />
+                  </ToolbarItem>
+                </ToolbarGroup>
+              </ToolbarContent>
+            </Toolbar>
+            <div className="pf-v5-u-mb-md">
+              <Table
+                aria-label="Simple table"
+                variant="compact"
+                borders
+                className="pf-v5-u-m-0 pf-v5-u-p-0"
+              >
+                {isBug ? (
+                  <Thead>
+                    <Tr>
+                      <Th className={issueTableColumnClass.issueKey}>Bug issue key</Th>
+                      <Th className={issueTableColumnClass.bugUrl}>URL</Th>
+                      <Th className={issueTableColumnClass.summary}>Summary</Th>
+                      <Th className={issueTableColumnClass.uploadDate}>Last updated</Th>
+                      <Th className={issueTableColumnClass.status}>Status</Th>
+                    </Tr>
+                  </Thead>
+                ) : (
+                  <Thead>
+                    <Tr>
+                      <Th className={issueTableColumnClass.issueKey}>CVE key</Th>
+                      <Th className={issueTableColumnClass.cveUrl}>URL</Th>
+                      <Th className={issueTableColumnClass.components}>Components</Th>
+                      <Th className={issueTableColumnClass.summary}>Summary</Th>
+                      <Th className={issueTableColumnClass.uploadDate}>Last updated</Th>
+                      <Th className={issueTableColumnClass.status}>Status</Th>
+                    </Tr>
+                  </Thead>
+                )}
+
+                {Array.isArray(filteredIssues) && filteredIssues.length > 0 && (
+                  <Tbody data-test="issue-table-body">
+                    {filteredIssues.map((issue, i) => (
+                      <Tr key={issue.key}>
+                        <Td className={issueTableColumnClass.issueKey}>{issue.key}</Td>
+                        <Td
+                          className={
+                            isBug ? issueTableColumnClass.bugUrl : issueTableColumnClass.bugUrl
+                          }
+                        >
+                          <Truncate content={issue.url} />
```

Comment:
URL field accepts any random text, shouldn't we add url validation while adding a bug fix info ?

<img width="1050" alt="Screenshot 2024-03-18 at 2 26 20 PM" src="https://github.com/openshift/hac-dev/assets/9964343/e0548d40-aa4b-45b1-b4f9-63576aa4ac3d">